build.gradle.kts

```diff
@@ -77,29 +85,23 @@ dependencies {
     implementation("io.insert-koin:koin-ktor:$koinVersion")
 
     // Test
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.9.1")
-    testImplementation("org.junit.vintage:junit-vintage-engine:5.9.0")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:$jupiterVersion")
+    testImplementation("org.junit.vintage:junit-vintage-engine:$jupiterVersion")
 
     testImplementation("com.willowtreeapps.assertk:assertk-jvm:0.25")
     testImplementation("org.apache.kafka:kafka-clients:$kafkaVersion")
 
     testImplementation("no.nav.helse:dusseldorf-test-support:$dusseldorfKtorVersion")
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
-    testImplementation("io.mockk:mockk:1.12.8")
-    testImplementation("io.ktor:ktor-server-test-host:$ktorVersion") {
+    testImplementation("io.mockk:mockk:1.13.1")
+    testImplementation("io.ktor:ktor-server-test-host-jvm:$ktorVersion") {
         exclude(group = "org.eclipse.jetty")
```

Comment:
bruker ktor junit 5 nå?

Reply:
Den drar in junit4 som postgres-testcontainers er avhengig av. Så går ikke o exkludere junit4 ur projektet.

Reply:
hvis postgres testcontainers migrerer over til junit 5 så må vi fortsette å exclude her da?